drizzlepac/adrizzle.py

```diff
@@ -856,26 +864,26 @@ def run_driz_chip(img,chip,output_wcs,outwcs,template,paramDict,single,
 
     # Merge appropriate additional mask(s) with DQ mask
     if single:
-        mergeDQarray(staticMaskName,dqarr)
+        mergeDQarray(staticMaskName, dqarr)
         if dqarr.sum() == 0:
             log.warning('All pixels masked out when applying static mask!')
     else:
-        mergeDQarray(staticMaskName,dqarr)
+        mergeDQarray(staticMaskName, dqarr)
 
         if dqarr.sum() == 0:
             log.warning('All pixels masked out when applying static mask!')
         else:
             # Only apply cosmic-ray mask when some good pixels remain after
             # applying the static mask
-            mergeDQarray(crMaskName,dqarr)
+            mergeDQarray(crMaskName, dqarr)
 
             if dqarr.sum() == 0:
                 log.warning('WARNING: All pixels masked out when applying '
                             'cosmic ray mask to %s' % _expname)
-        updateInputDQArray(chip.dqfile,chip.dq_extn,chip._chip,
+        updateInputDQArray(chip.dqfile ,chip.dq_extn, chip._chip,
                            crMaskName, paramDict['crbit'])
```

Comment:
Line 883 (eyes are gaga): space added in wrong place - remove after chip.dqfile and put after comma.

Reply:
Corrected in b11537b4